### ISSUE TITLE
Orange PI one hangs during boot

### ISSUE THREAD

**Issue by kenthinson** [jernejsk/OpenELEC-OPi2#82]
Orange PI one hangs during boot at the OpenELEC logo. Using the newest image. Is ssh turned on by default so I can get logs?

**Comment by jernejsk** [comment]
No, if it would be, it would pose a security threat, since you can't change the password on OpenELEC. It wouldn't do any good either since you can't run user space program during boot. In order to get kernel output you need usb to serial adapter and connect it to the "debug" port on board.

Anyway, I stopped actively working on this project. If you find the solution, you can make a pull request. If the solution is reasonable, I will merge it.
